doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
+      </p>
+      <p>
+        The GitHub release shall be named <code>rYYYY-pubState</code>, with a tag assigned to the release that matches the release name. If creating a CD, itterate the <code>pubState</code>, starting with <code>1</code>. 
+      </p>
+      <p class="example">
+        <code>r2023-fcd</code> or <code>r2023-cd2</code>
+      </p>
+      <p class="note">
+        The itteration of CD is to account for multiple possible PCD periods. 
+      </p>
+      <p>
+        When creating the next update on the repo, update <code>latestEditionTag</code> in the <code>.smpte-build.json</code> file to the value equal to the tag name of this latest release. This will trigger the tooling to create a link for <code>Redline to most recent edition</code> during any PRs and merges to the main branch. 
+      </p>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing releases. 
+      </p>
+
+    </section>
+
+    <section id="sec-sample-workflow">
+      <h3>Sample Workflow </h3>
+      <p>
+        <a href="#tab-sample-workflow"></a> shows a sample workflow would be the steps for an Engineering Document going through the an initial draft or revision and the balloting process. 
+      </p>
+      <table id="tab-sample-workflow">
+        <caption>Sample workflow</caption>
+        <thead>
+          <tr>
+            <th>Workflow Step</th>
+            <th>Action(s) on document and/or GitHub repo</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>(1) - Project is approved</td>
+            <td>
+              <ol>
+                <li>If document is an intial draft, skip to Workflow Step (2)</li>
+                <li>If document is a revision, skip to Workflow Step (3)</li>
+              </ol>
+            </td>
+          </tr>
+          <tr>
+            <td>(2) - Project is a document initial draft</td>
+            <td>
+              <ol>
+                <li>Follow the steps in <a href="#sec-quick-start"></a> to create repo.</li>
+                <li>A new branch is created from <code>Main</code> called <code>r2023-inititial-draft</code>.</li>
+                <li>Template document is updated in <code>r2023-inititial-draft</code> branch to <code>&lt;meta itemprop="pubState" content="draft" /&gt;</code> and to <code>&lt;meta itemprop="pubStage" content="WD" /&gt;</code>.</li>
+                <li>Document editor creates initial version of the document.</li>
+                <li>A PR is created named <code>2023 Initial Draft</code> from the <code>r2023-inititial-draft</code> branch on GitHub. </li>
+                <li>Skip to Workflow Step (4)</li>
+              </ol>
+            </td>
+          </tr>
+          <tr>
+            <td>(3) - Project is a document revision</td>
+            <td>
```

Comment:
Need to update `.smpte-build.json`?

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96